Sources/Recoverable/Recoverable.swift

```diff
@@ -18,61 +18,61 @@ extension UIView: Recoverable {
         get { return ao_get(pkey: &ViewAssociatedKeys.viewState) as? RecoverableViewState }
         set { ao_setOptional(newValue, pkey: &ViewAssociatedKeys.viewState) }
     }
-    
+
     @objc func saveViewState() {
         viewState = RecoverableViewState(view: self)
     }
-    
+
     @objc func recoverViewState(forced: Bool) {
         guard let safeViewState = viewState else { return }
-        
+
         startTransition { [weak self] in
             self?.layer.cornerRadius = safeViewState.cornerRadius
             self?.layer.masksToBounds = safeViewState.clipToBounds
-            
+
             if safeViewState.backgroundColor != self?.backgroundColor || forced {
                 self?.backgroundColor = safeViewState.backgroundColor
             }
         }
     }
 }
 
-extension UILabel{
+extension UILabel {
     var labelState: RecoverableTextViewState? {
         get { return ao_get(pkey: &ViewAssociatedKeys.labelViewState) as? RecoverableTextViewState }
         set { ao_setOptional(newValue, pkey: &ViewAssociatedKeys.labelViewState) }
     }
-    
+
     override func saveViewState() {
         super.saveViewState()
         labelState = RecoverableTextViewState(view: self)
     }
-    
+
     override func recoverViewState(forced: Bool) {
         super.recoverViewState(forced: forced)
         startTransition { [weak self] in
             self?.textColor = self?.labelState?.textColor
-            self?.text = self?.labelState?.text
+            self?.attributedText = self?.labelState?.attributedText
         }
     }
 }
 
-extension UITextView{
+extension UITextView {
     var textState: RecoverableTextViewState? {
         get { return ao_get(pkey: &ViewAssociatedKeys.labelViewState) as? RecoverableTextViewState }
         set { ao_setOptional(newValue, pkey: &ViewAssociatedKeys.labelViewState) }
     }
-    
+
     override func saveViewState() {
         super.saveViewState()
         textState = RecoverableTextViewState(view: self)
     }
-    
+
     override func recoverViewState(forced: Bool) {
         super.recoverViewState(forced: forced)
         startTransition { [weak self] in
             self?.textColor = self?.textState?.textColor
-            self?.text = self?.textState?.text
+            self?.attributedText = self?.textState?.attributedText
```

Comment:
what happen with the text?

Reply:
We use `attributedText` on our labels, if we set it and show the skeletons, the recoverable will replace the attributes with just plain `text`, removing the styles.

Reply:
@Juanpe thoughts here?